src/main/java/ladder/controller/LadderController.java

```diff
@@ -1,27 +1,34 @@
 package ladder.controller;
 
+import java.util.Map;
+import ladder.dto.LineDto;
 import ladder.model.Ladder;
 import ladder.model.Players;
+import ladder.service.LadderGame;
 import ladder.view.InputView;
 import ladder.view.OutputView;
 
 import java.util.List;
 
 public class LadderController {
-    private Players ladderPlayers;
     private Ladder ladder;
+    private LadderGame ladderGame;
 
     public void start() {
         init();
-        printResult();
+        showLadder();
+        showReward(readLookupTarget());
     }
 
     private void init() {
-        ladderPlayers = Players.from(readPlayerNames());
+        Players ladderPlayers = Players.from(readPlayerNames());
 
         int height = readLadderHeight();
         int width = ladderPlayers.getSize();
+
+        List<String> rewards = readRewards();
         ladder = Ladder.of(height, width);
+        ladderGame = LadderGame.from(ladderPlayers, rewards, ladder);
```

Comment:
```suggestion
        int height = readLadderHeight();
        int width = ladderPlayers.getSize();
        ladder = Ladder.of(height, width);
        
        List<String> rewards = readRewards();
        ladderGame = LadderGame.from(ladderPlayers, rewards, ladder);
```
step 1 에서 남겼던 리뷰처럼 이번에도 초기화와 사용처를 가깝게 배치할 수 있을 것 같네요 :)

Reply:
이제는 Ladder 필드가 사라지면서 `ladder` 역시 초기화 구문이 돼 버렸네요ㅎㅎ
앞으로도 꼼꼼히 체크하겠습니다!